internals/daemon/daemon.go

```diff
@@ -327,10 +379,10 @@ func logit(handler http.Handler) http.Handler {
 				r.URL.Path == "/v1/health")
 		if !skipLog {
 			if strings.HasSuffix(r.RemoteAddr, ";") {
-				logger.Debugf("%s %s %s %s %d", r.RemoteAddr, r.Method, r.URL, t, ww.status())
-				logger.Noticef("%s %s %s %d", r.Method, r.URL, t, ww.status())
+				logger.Debugf("%s %s %s %s %d (source: %s)", r.RemoteAddr, r.Method, r.URL, t, ww.status(), connSource.String())
+				logger.Noticef("%s %s %s %d (source: %s)", r.Method, r.URL, t, ww.status(), connSource.String())
```

Comment:
Since `APIRequestSource` implements `Stringer`, we don't need this! (:

```suggestion
				logger.Debugf("%s %s %s %s %d (source: %s)", r.RemoteAddr, r.Method, r.URL, t, ww.status(), connSource)
				logger.Noticef("%s %s %s %d (source: %s)", r.Method, r.URL, t, ww.status(), connSource)
```

Reply:
Fixed, thank you. Resolving.